modules/gis/src/lib/binary-to-geojson.ts

```diff
@@ -20,34 +20,36 @@ import type {
 // Note:L We do not handle GeometryCollection, define a limited Geometry type that always has coordinates.
 // type FeatureGeometry = Point | MultiPoint | LineString | MultiLineString | Polygon | MultiPolygon;
 
+type BinaryToGeoJsonOptions = {
+  type?: BinaryGeometryType;
+  globalFeatureId?: number;
+};
+
 /**
  * Convert binary geometry representation to GeoJSON
  * @param data   geometry data in binary representation
- * @param type   Input data type: Point, LineString, or Polygon
- * @param format Output format, either geometry or feature
+ * @param options
+ * @param options.type  Input data type: Point, LineString, or Polygon
+ * @param options.featureId  Global feature id. If specified, only a single feature is extracted
  * @return GeoJSON objects
  */
-export function binaryToGeoJson(
+export function binaryToGeoJSON(
   data: BinaryGeometry | BinaryFeatures,
```

Comment:
I thought we said we would only support `data: BinaryFeatures` and restrict the return type to `Feature[] | Feature` now that we have `parseGeometry`

Reply:
In fact, unless I'm missing something, if we take that line of thinking to its' logical conclusion, perhaps we want to remove the `isHeterogeneousType` legacy check and remove the `type` from `BinaryToGeoJsonOptions` as it just seems to be a roundabout way to invoking `parseGeometry`

Reply:
I'm happy to restrict `binaryToGeoJson` to just `BinaryFeatures` and use `binaryToGeometry` for `BinaryGeometry` type data.

Reply:
I've restricted `binaryToGeoJson` to just `BinaryFeatures` but have left the `isHeterogeneousType` check for back-compatibility